metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures datadog components'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.1.0'
+version          '3.1.1'
```

Comment:
version bump should be removed from PR

Reply:
Yeah, we're using this from the QA env, so I needed the bump. But we'll remove that before merging.

Reply:
Let's remove this and merge to `master`!